services/postgres/alembic/versions/2020-09-30T123806_add_label_column_to_flow_runs.py

```diff
@@ -0,0 +1,48 @@
+"""
+Add label column to flow runs
+
+Revision ID: 24f10aeee83e
+Revises: 850b76d44332
+Create Date: 2020-09-30 12:38:06.915340
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB, UUID
+
+
+# revision identifiers, used by Alembic.
+revision = "24f10aeee83e"
+down_revision = "850b76d44332"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "flow_run", sa.Column("labels", JSONB, nullable=False, server_default="[]")
+    )
+    op.execute(
```

Comment:
Just for future reference do you mind adding a small comment here - something like "for runs scheduled at the time this migration is executed, we want to make sure they have the new logic."

Reply:
Updated